Examples/HttpsExampleServer/bin/Debug/netcoreapp2.1/HttpsExampleServer.deps.json

```diff
@@ -0,0 +1,302 @@
+{
```

Comment:
Please remove all local build artificats (/bin and /obj folders). The other examples have a `.gitignore` that they shouldn't get checked in. Should be the same here.